app/views/content/life-as-a-teacher/biology/_article.html.erb

```diff
@@ -0,0 +1,97 @@
+    <p>You could have the opportunity to teach or support physics and biology lessons throughout key stages 3 and 4.</p>
+    <p>When you teach 11 to 14 year olds (key stage 3), you'll cover themes from the 3 science disciplines of biology, chemistry and physics.</p>
+    <p>The biology themes are:</p>
+    <ul>
+      <li>structure and function of living organisms</li>
+      <li>material cycles and energy</li>
+      <li>interactions and interdependencies</li>
+      <li>genetics and evolution</li>
+    </ul>
+  </section>
+
+  <section class="col col-720">
+    <p>Themes you'll cover when you teach 14 to 16 year olds (key stage 4) will be more specific to biology, and include:</p>
+    <ul>
+      <li>cell biology</li>
+      <li>transport systems</li>
+      <li>health, disease and the development of medicines</li>
+      <li>coordination and control</li>
+      <li>photosynthesis</li>
+      <li>ecosystems</li>
+      <li>evolution, inheritance and variation</li>
+    </ul>
+    <p><a href="https://www.thenational.academy/teachers/curriculum/science-secondary/units">Explore what teaching a key stage 4 biology lesson would be like</a>.</p>
+    <p>As a secondary school biology teacher, you'll teach students aged 11 to 18. This means you'll always be expanding your knowledge and approach to meet students' needs. Once you have qualified teacher status (QTS) you can also teach any other subject, as long as you can show you have enough expertise.</p>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h2 class="heading--box-blue">Start your journey into teaching biology</h2>
+    <h3>Check your qualifications</h3>
+    <p>To train to teach in secondary schools in England, you’ll need:</p>
+    <ul>
+      <li>GCSEs at grade 4 (C) or above in English and maths</li>
+      <li>a bachelor’s degree in any subject</li>
+    </ul>
+    <p>Having relevant A levels can show your subject knowledge, if you do not have a degree in biology.</p>
+    <p>Find out more about what <a href="/train-to-be-a-teacher/qualifications-you-need-to-teach">qualifications you need to train to teach</a>.</p>
+    <%= render Content::ExpanderComponent.new(
+      title: "check your qualifications",
+      text: "If you have qualifications from outside the UK, you'll need to show that they meet the standards set for teacher training in England.",
+      link_title: "You can get help comparing English and international qualifications.",
+      link_url: "/non-uk-teachers/non-uk-qualifications",
+    ) %>
+  </section>
+
+  <section class="col col-space-m col-720">
+    <h3>Fund your teacher training</h3>
+    <p><a href="/funding-and-support/scholarships-and-bursaries">Tax-free bursaries of <%= v :bursaries_postgraduate_computing %> or scholarships of <%= v :scholarships_computing %></a> are available for eligible trainee computing teachers.</p>
```

Comment:
this info is about computing? also might need to update the non-UK expander to say that there is funding for physics for non-uk?

Reply:
changed to biology - Sarah to add content regarding non-uk if appropriate

Reply:
@Sarah-DfE